components/breadcrumbs/demo/breadcrumbs.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
+	<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+	<script type="module">
+		import '../../demo/demo-page.js';
+		import '../breadcrumbs.js';
+	</script>
+	<style>
+		.align-item {
+			color: var(--d2l-color-tungsten);
+			font-size: 0.7rem;
+		}
+		.align-item.bullet::before {
+			color: var(--d2l-color-tungsten);
+			content: '\002022';
+			margin-left: 5px;
+			margin-right: 5px;
+		}
+	</style>
```

Comment:
It looks like the styles in this block are unused so you should be good to remove it.

Reply:
Removed ✔️